content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md

```diff
@@ -40,7 +44,9 @@ The .NET Tracer can instrument the following libraries automatically:
 | MongoDB                         | `MongoDB.Driver.Core` 2.1.0+                                            | `MongoDb`            |
 | PostgreSQL                      | `Npgsql` 4.0+                                                           | `AdoNet`             |
 
-**Note:** The ADO.NET integration instruments calls made through the `DbCommand` abstract class or the `IDbCommand` interface, regardless of the underlying implementation. It also instruments direct calls to `SqlCommand`.
+<div class="alert alert-warning">
+<strong>Note:</strong> The ADO.NET integration instruments calls made through the <code>DbCommand</code> abstract class or the <code>IDbCommand</code> interface, regardless of the underlying implementation. It also instruments direct calls to <code>SqlCommand</code>.
+</div>
 
 Don’t see your desired frameworks? Datadog is continually adding additional support. [Check with the Datadog team][3] for help.
```

Comment:
We should keep this line in sync between the dotnet-core and dotnet-framework pages

Reply:
Additionally, they should be the same on both pages with regarding to placement before/after the table. I'd vote to keep it after